doc/bitcoin-conf.md

```diff
@@ -60,3 +60,10 @@ Linux | `$HOME/.bitcoin/` | `/home/username/.bitcoin/bitcoin.conf`
 macOS | `$HOME/Library/Application Support/Bitcoin/` | `/Users/username/Library/Application Support/Bitcoin/bitcoin.conf`
 
 You can find an example bitcoin.conf file in [share/examples/bitcoin.conf](../share/examples/bitcoin.conf).
+
+## Environment variables
+
+Name | Option equivalent
+-- | --
+BITCOIN_DATADIR | -datadir
+BITCOIN_CONF	| -conf
```

Comment:
This complicates the init process and argument parsing. I am ~0 on the overall pull, but at the very least, this should include documentation on the precedence. Tests wouldn't hurt either.

Reply:
It doesn't really complicates the init process. It just checks if a specific argument is set. If not it will use the envvar

Reply:
@MarcoFalke Updated, please check